snippets/en/_includes/self-managed-version-requirements.mdx

```diff
@@ -0,0 +1,7 @@
+| Software     | Minimum version                              |
+| ------------ | -------------------------------------------- |
+| Kubernetes   | v1.29                                        |
```

Comment:
I've been testing and using W&B on the latest versions of Kubernetes without issues, so I think we should recommend the supported versions rather than `v1.29`.
I suggest we recommend starting at least from `v1.32`
https://kubernetes.io/releases/patch-releases/#non-active-branch-history